autoload/lsp/omni.vim

```diff
@@ -248,8 +248,12 @@ function! lsp#omni#default_get_vim_completion_item(item, ...) abort
     endif
 
     if has_key(a:item, 'documentation')
-        if type(a:item['documentation']) == type('')
+        if type(a:item['documentation']) == type('') " field is string
             let l:completion['info'] .= a:item['documentation']
+        elseif type(a:item['documentation']) == type({}) && 
+                    \ has_key(a:item['documentation'], 'value')
+            " field is MarkupContent (string or markdown, we don't care)
+            let l:completion['info']['value'] .= a:item['documentation']
```

Comment:
Will this not fail because 'info' is a string?

Reply:
Ah, I think you want to flip the `value` to the other side?

Reply:
:man_facepalming: